parsing/blocks/blockparser/common.go

```diff
@@ -35,6 +35,11 @@ func convertIndexerTxsToParserTxs(txs []indexer.Transaction) []Transaction {
 }
 
 func convertIndexerBlockToParserBlock(block indexer.Block, hash string) Block {
+	// TODO: Check why we discard block.Hash.
+	// TODO: Check if we can drop "validators" field - is it required?
+	// TODO: Check if we should also add miniBlocksHashes field.
+	// TODO: Check why MiniBlocks field is nil.
```

Comment:
is this structure supposed to mirror the block header? 
or it should have the entire block data (header + body) 
then we can decide if we add the miniblocks or just the miniblock headers.

Reply:
Adjusted the schema (simplified), to handle hyperblocks.